packages/calling/src/Contacts/types.ts

```diff
@@ -69,17 +68,78 @@ export type ContactResponse = {
   message: string | null;
 };
 
+/**
+ * Interface for Contacts Module.
+ * This includes the APIs that allows to fetch, create and update the contacts and groups.
+ *
+ */
 export interface IContacts {
-  getSDKConnector: () => ISDKConnector;
-  getContacts: () => Promise<ContactResponse>;
-  createContactGroup: (
+  /**
+   * This API is used to fetch the list of contacts and groups for a user.
+   *
+   * Example
+   * ```javascript
+   * const contactsResponse = await contactClient.getContacts();
+   * ```
+   *
+   * The contactsResponse object will have the list of contacts and groups
+   * Each contact object will have properties as mentioned in Contact
+   * Each group object will have properties as mentioned in ContactGroup
+   *
+   * Example - Contact - https://github.com/webex/webex-js-sdk/wiki/Calling-Contacts#contact
+   *
+   * Example - ContactGroup - https://github.com/webex/webex-js-sdk/wiki/Calling-Contacts#contactgroup
```

Comment:
Once again, better to export them rather than pointing to a different document. What if we change here in SDK and miss to change in the Wiki? One of these places at the least should show correct info

Reply:
Discussed this with Sreekanth and decided to export the types and pointed the link to it wherever needed. The description is not present for each property in these types. Will be added in the next PR

Reply:
I will create one if needed else we can work on it as adhoc activity

Reply:
@mkesavan13 Added comment in this Jira: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-428325 to take care of this change